src/redux/categories/categoriesSlice.js

```diff
@@ -11,5 +11,5 @@ const categoriesSlice = createSlice({
   },
 });
 
-export const { addBook, removeBook } = categoriesSlice.actions;
+export const categoriesActions = categoriesSlice.actions;
 export default categoriesSlice.reducer;
```

Comment:
- [ ] **Good job so far 👏**  however, I think it would be better to adjust the button for checking the status
to use a function like ```checkCategoriesStatus``` made in the component and dispatches the ```underConstruction``` action creator, Currently the button is not functioning 😎😎😎

example:
![image](https://user-images.githubusercontent.com/102757126/222779327-10956284-ac97-4088-8120-4a45dc3d7eff.png)

Reply:
# I will implement the button. But for now I have put this in under constructions.
😄 Thanks for the suggestion.